spec/spec_helper.rb

```diff
@@ -0,0 +1,19 @@
+require 'rack/test'
+require 'rspec'
+
+ENV['RACK_ENV'] = 'test'
+
+require File.expand_path '../../whedon.rb', __FILE__
+
+module RSpecMixin
+  include Rack::Test::Methods
+  def app() Sinatra::Application end
+end
+
+RSpec.configure do |config|
+  config.before(:each) do
+    stub_request(:any, /api.github.com/).to_rack(FakeGitHub)
```

Comment:
This is meant to intercept any outgoing requests and pass them to the `FakeGitHub` app (see https://robots.thoughtbot.com/how-to-stub-external-services-in-tests)